platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh

```diff
@@ -15,11 +15,14 @@ fi
 
 SSD_UPGRADE_LOG="$SSD_FW_UPGRADE/upgrade.log"
 
+iSMART="/usr/local/bin/iSMART_64"
+iSMART_OPTIONS="-d /dev/sda"
+iSMART_CMD=`$iSMART $iSMART_OPTIONS`
 SMART_CMD=`smartctl -a /dev/sda`
 
-SSD_FW_VERSION=$(echo "$SMART_CMD" | grep "Firmware Version" | awk '{print $NF}')
+SSD_FW_VERSION=$(echo "$iSMART_CMD" | grep "FW Version" | awk '{print $NF}')
 SSD_FW_VERSION=${SSD_FW_VERSION,,}
-SSD_MODEL=$(echo "$SMART_CMD" | grep "Device Model" | awk '{print $NF}')
+SSD_MODEL=$(echo "$iSMART_CMD" | grep "Model" | awk '{print $NF}')
```

Comment:
Why echo'ing the command line instead of executing it? Not sure I follow this line and line 23.

Reply:
We calling the command in line 20 and save the output of the command in iSMART_CMD variable.
Thats why using echo in line 23

Reply:
ah I missed the ``. :-)